### DISCUSSION TITLE
Network request to www.gstatic.com

### DISCUSSION THREAD

**Discussion by ckr0** [jellyfin/jellyfin#7328] [General]
Hi why does this app makes a request to www.gstatic.com. Is it by design? and why it happens only on Chrome browsers and non-chrome browsers with chrome as user agent? It is only triggered  after login. This happened on multiple devices and device types.

This is a completely offline private network jellyfin installation. 

Someting to do with DRM?

**Comment by cvium** [comment]
Chromecast https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1
